package/MDAnalysis/lib/nsgrid.pyx

```diff
@@ -90,6 +90,10 @@ DEF XZ = 6
 DEF YZ = 7
 DEF ZZ = 8
 
+# Cube root of the maximum size of a 32 bit signed integer. If the system is divided into more
```

Comment:
Not necessarily - at that limit, smaller grids are only potentially helpful if your total atom count is staggeringly large. In the case used to reproduce this bug, the grid count is quite excessive. I took a look at the documentation - it doesn't get into the fine details of how grids/grid sizes are selected, that info is hidden from the user. I can add a section on it, but I think it would probably be a bit too much detail.

Reply:
Added versionchanged notes